spec/request/ppiu_request_spec.rb

```diff
@@ -93,5 +93,27 @@
         end
       end
     end
+
+    context 'with a 500 server error type pertaining to potential fraud' do
+      it 'should return a service error response' do
```

Comment:
Perhaps add in `, :aggregate_failures` for these multiples expectation `it` blocks?

Reply:
>_< I never think of this.